fastlane/actions/rocket.rb

```diff
@@ -0,0 +1,64 @@
+module Fastlane
+  module Actions
+    class RocketAction < Action
+      def self.run(params)
+        puts "
+        ____
+       /    \\
+      |      |
+      |      |
+      |      |
+       \\____/
+       |    |
+       |    |
+       |    |
+       |____|
+      {|    |}
+       |    |
+       |    |
+       | F  |
+       | A  |
+       | S  |
+       | T  |
+       | L  |
+       | A  |
+      /| N  |\\
```

Comment:
Clearly the rocket is a cylinder, and the writing is still in the center. This image was just made at an angle.